.code-samples.meilisearch.yaml

```diff
@@ -1373,7 +1373,7 @@ search_parameter_reference_locales_1: |-
   -X POST 'http://localhost:7700/indexes/INDEX_NAME/search' \
   -H 'Content-Type: application/json' \
   --data-binary '{
-    "q": "進撃の巨人",
+    "q": "QUERY TEXT IN JAPANESE",
```

Comment:
why do we change? what was the issue with the previous example?

Reply:
To make this sample consistent with the pattern we've been following with new code samples, where `UPPERCASE TEXT` signals placeholder content. A bit like the URL a couple lines above: `http://localhost:7700/indexes/INDEX_NAME/search`.